mm2src/mm2_bitcoin/chain/src/block_header.rs

```diff
@@ -197,6 +203,15 @@ impl Deserializable for BlockHeader {
         }
         let previous_header_hash = reader.read()?;
         let merkle_root_hash = reader.read()?;
+
+        // This is needed to deserialize coin like LBC correctly.
+        let claim_trie_root = if version == QTUM_BLOCK_HEADER_VERSION && *reader.coin_name() == Some("LBC".to_string())
```

Comment:
It's better to create `LBC_BLOCK_HEADER_VERSION` equal to `QTUM_BLOCK_HEADER_VERSION` so two coins protocols are not mixed up here.

Reply:
This is actually the no soft fork block version https://bitcoincore.org/en/2016/06/08/version-bits-miners-faq/, so it can be named to `BIP9_NO_SOFT_FORK_BLOCK_HEADER_VERSION` as a more generic naming for `Qtum` and `LBC` and any other coin that uses this version in the future.

Reply:
Yeah, it's a good note 🙂